docs/quick-start.md

```diff
@@ -86,11 +86,12 @@ edit:add-var y~ {|@argv|
 
 ```powershell
 function y {
-    $tmp = [System.IO.Path]::GetTempFileName()
+    $tmp = New-TemporaryFile | foreach FullName
     yazi $args --cwd-file="$tmp"
     $cwd = Get-Content -Path $tmp -Encoding UTF8
     if (-not [String]::IsNullOrEmpty($cwd) -and $cwd -ne $PWD.Path) {
-        Set-Location -LiteralPath ([System.IO.Path]::GetFullPath($cwd))
+        $path = Get-Item -LiteralPath $cwd -ea Stop
+        Set-Location -LiteralPath $path.FullName
```

Comment:
nit: is it possible to merge these two into one line?

Reply:
`Resolve-Path` seems to resolve symlinks, which is unintended - if the user exits Yazi in `C:\foo`, it should enter `C:\foo` itself, not the directory to which `C:\foo` points.